frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewMetaData/EnvironmentVisibilityMenu/hooks/useEnvironmentVisibility.ts

```diff
@@ -0,0 +1,39 @@
+import { useLocalStorageState } from 'hooks/useLocalStorageState';
+import { usePlausibleTracker } from 'hooks/usePlausibleTracker';
+import { createLocalStorage } from 'utils/createLocalStorage';
+
+// Reading legacy value be safely refactored out in a next version - related to `flagOverviewRedesign` flag
```

Comment:
Is this missing a word? I'm not sure I understand the comment (but maybe that's not important)

```suggestion
// Reading legacy value will be safely refactored out in a next version - related to `flagOverviewRedesign` flag
```